src/applications/gi/containers/GiBillApp.jsx

```diff
@@ -89,8 +91,11 @@ export class GiBillApp extends React.Component {
       content = this.props.children;
     }
 
+    const banner = environment.isProduction() ? '' : <Covid19Banner />;
+
     return (
       <div className="gi-app">
+        {banner}
```

Comment:
Can you drop the `banner` and do `!environment.isProduction() && <Covid19Banner />`?

Reply:
sure